src/storage/heap_file.c

```diff
@@ -21630,6 +21771,10 @@ heap_update_bigone (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context, b
       or_mvcc_set_log_lsa_to_record (context->recdes_p, logtb_find_current_tran_lsa (thread_p));
     }
 
+/* SUPPLEMENT_UPDATE UNDO LSA 
```

Comment:
Which code is this comment for?

Reply:
Oh.. this is just trash comment during implementation. I will remove it